### DISCUSSION TITLE
Github action banned.

### DISCUSSION THREAD

**Discussion by HELLSNAKES** [community/community#50982] [Actions]
### Select Topic Area

Question

### Body

My account has been banned from using action, how can i use it again and how long does it take for me to get unblocked ?

![image](https://user-images.githubusercontent.com/63496374/227687017-27742c31-7dde-4d6e-adfb-d4751c1e84f6.png)

**Comment by airtower-luna** [comment]
As the message says, you'll have to ask support, us other users can't help there. Use the link in the message.
